components/00-design-tokens/colors.config.yml

```diff
@@ -1,60 +1,57 @@
-context: 
+notes: "some notes"
+context:
     colorset:
-        - label: color-primary
-          color: "#0071bc"
-        - label: color-primary-darker
-          color: "#205493"
-        - label: color-primary-darkest
-          color: "#112e51"
-        - label: color-primary-alt
-          color: "#02bfe7"
-        - label: color-primary-alt-dark
-          color: "#00a6d2"
-        - label: color-primary-alt-darkest
-          color: "#046b99"
-        - label: color-primary-alt-light
-          color: "#9bdaf1"
-        - label: color-primary-alt-lightest
-          color: "#e1f3f8"
-        - label: color-secondary
-          color: "#e31c3d"
-        - label: color-secondary-dark
-          color: "#cd2026"
-        - label: color-secondary-darkest
-          color: "#981b1e"
-        - label: color-secondary-light
-          color: "#e59393"
-        - label: color-secondary-lightest
-          color: "#f9dede"
-        - label: color-white
-          color: "#ffffff"
-        - label: color-black
-          color: "#000000"
-        - label: color-base
-          color: "#212121"
-        - label: color-gray-dark
-          color: "#323a45"
-        - label: color-gray
-          color: "#5b616b"
-        - label: color-gray-medium
-          color: "#757575"
-        - label: color-gray-light
-          color: "#aeb0b5"
-        - label: color-gray-lighter
-          color: "#d6d7d9"
-        - label: color-gray-lightest
+        - label: black
```

Comment:
Should we continue to namespace these variables (even with `$c-`?)

On the one hand, color names are pretty distinct. On the other, we might need namespacing for some variables (like margins vs. font sizes), and it would be good to keep things consistent.